Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM jenkins:2.7.1
+
+ENV MESOS_VERSION=0.28.1\*
+
+# Mesos plugin needs mesos binary to be present
+USER root
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
+    sh -c "echo deb http://repos.mesosphere.io/debian jessie main > /etc/apt/sources.list.d/mesosphere.list" && \
+    apt-get update && \
+    apt-get install -y mesos=$MESOS_VERSION && \
+    rm -rf /var/lib/apt/lists/*
+
+USER jenkins
+
+# Disable executors
+COPY src/main/resources/docker/executors.groovy /usr/share/jenkins/ref/init.groovy.d/executors.groovy
```

Comment:
Could you put this file under src/main/docker instead? Otherwise it will end up in the packaged hpi for no good reason.